docs/dev/test-configuration-options.md

```diff
@@ -471,29 +471,23 @@ capabilities.setCapability("appium:orientation", "LANDSCAPE");
 
 <p><small>| OPTIONAL | BOOLEAN | <span className="sauceDBlue">Virtual and Real Devices</span> |</small></p>
 
-**For Virtual Devices:**
-
-Set `noReset` to `true` when you execute multiple tests on a single virtual device to keep the state it is in between tests.
+Prevents the device to be reset before the session startup if set to `true`. This means that the application under test is not going to be terminated neither its data cleaned. This capability behaves differently across virtual and real devices.
 
 **For Real Devices:**
 
 Set `noReset` to `true` to keep a device allocated to you during the device cleaning process, as described under [`cacheId`](#`cacheId`), allowing you to continue testing on the same device. Default value is `false`. To use `noReset`, you must pair it with `cacheId`.
 
-**Specifics for Android Virtual and Real Devices:**
+**For Virtual Devices:**
 
+This capability will have no effect on Sauce Labs virtual devices, it will only work on local Android emulators / iOS simulators. For local executions you will likely only have one device available, in Sauce Labs you will have a pool of devices available, depending on your concurrency. Each session will start a new clean session which will make this capability redundant.
```

Comment:
```suggestion
This capability will have no effect on Sauce Labs virtual devices, it will only work on local Android Emulators/iOS Simulators. For local executions, you will likely only have one device available, in Sauce Labs you will have a pool of devices available depending on your concurrency. Each session will start a new clean session which will make this capability redundant.
```